templates/careers/index.html

```diff
@@ -18,80 +20,108 @@
   <img src='https://pubads.g.doubleclick.net/activity;xsp=5024608;ord=1?' width=1 height=1 border=0>
 </noscript>
 
-<section class="p-strip--image is-dark is-deep" style="background-image: url(https://assets.ubuntu.com/v1/aebc1742-01_suru2_dark_2K.png); background-position: 100% 55%; background-size: 180%; background-repeat: repeat-x;">
+<section class="p-section">
+  <div class="u-fixed-width">
+    <nav class="p-breadcrumbs u-sv1" aria-label="Breadcrumbs">
+      <ol class="p-breadcrumbs__items p-text--x-small-capitalised">
+          <li class="p-breadcrumbs__item ">Careers</li>
+      </ol>
+    </nav>
+    <h1 class="p-text--x-small-capitalised u-hide">Careers</h1>
+    <hr class="p-rule">
+  </div>
   <div class="row">
-    <div class="col-8 col-start-large-4">
-      <h1 class="p-heading--2">Make a difference in open source</h1>
+    <div class="col-9 col-start-large-4">
+      <div class="p-block">
+      <h2 class="u-no-margin--bottom p-heading--1">Passionate about open source?</h2>
+        <h3 class="p-heading--2">So are we. Be where the cutting edge <br class="u-hide--small u-hide--medium"/>is established.</h3>
+      </div>
       {% if vacancies %}
         {% include "careers/_search.html" %}
       {% endif %}
     </div>
   </div>
+  <div class="u-fixed-width u-no-padding u-hide--medium u-hide--small">
+    <div class="p-suru25light"></div>
+  </div>
 </section>
 
 {% if departments_overview %}
-<section class="p-strip--light">
-  <div class="u-fixed-width">
-    <h2 class="p-text--x-small-capitalised u-align-text--x-small-to-default">Departments</h2>
-    <hr>
-  </div>
+<section class="p-section">
   <div class="row">
-    {% for dept in departments_overview %}
-    <div class="col-3 col-medium-2 col-small-2">
-      <div class="p-heading-icon--small">
-        <div class="p-heading-icon__header">
-          <img src="https://assets.ubuntu.com/v1/{{dept.icon}}" class="p-heading-icon__img" style="width: 32px; height: 32px; position: relative; top: 7px;" alt="">
-          <p class="p-heading-icon__title">
-            <a href="/careers/{{dept.slug}}"><strong>{{ dept.name }}</strong></a><br />
-            <span>{{ dept.count }} roles</span>
-          </p>
+    <hr class="p-rule">
+    <div class="col-2 col-medium-12 p-block">
+      <h2 class="p-text--x-small-capitalised">Departments</h2>
+    </div>
+    <div class="col-12 col-medium-4">
+      <div class="row">
+        {% for dept in departments_overview %}
+        <div class="col-2 col-medium-12 col-small-12 p-department-cards__container">
+          <a href="/careers/{{dept.slug}}" style="text-decoration: none">
+          <div class="p-department-cards__card">
+            <div>
+              <img src="https://assets.ubuntu.com/v1/{{dept.icon}}" class="p-heading-icon__img" style="width: 32px; height: 32px; position: relative; top: 7px;" alt=""><br class="u-hide--medium u-hide--small" />
```

Comment:
Quite a lot of inline CSS to position the image. Should it be in CSS class name?